_posts/2015-11-09-scrum-basics-2.md

```diff
@@ -0,0 +1,118 @@
+
+The most important task for the Development Team, also in Scrum, is to deliver.
+Just in Scrum they can tell themselves, how much time they need to deliver something, which gives them the freedom to achieve the big goal of the Scrum Framework: to deliver quality products.
+
+# Product Owner
+
+{% image blog-scrum2-productowner.jpg 30% left %}{% endimage %}
+
+The Product Owner is a very important role in Scrum.
+I’ve heard trainers say that a Scrum Team can rise or fall with their PO.
+The speciality in Scrum is, that the PO is not the boss of the Development Team, but part of the team.
+This is not easy, especially for POs who have worked as Project Managers in the past.
+In Scrum they are a lateral leader.
+
+However, they do decide what the team is going to develop next.
+They are setting up the Product Backlog and decide the business value of each ticket and with that the ticket order.
+That means, the PO is responsible for the Return of Investment (ROI).
+They decide if fixing a bug is more important than developing a new feature.
+They spend a lot of time maintaining the Product Backlog, which means creating new tickets and prioritising them as well as keeping the backlog up to date.
+Several techniques exist of how to prioritise a Backlog.
+The technique to be used mainly depends on the product, the team is working on.
+
+To create the tickets the PO is also responsible for gathering all the requirements as well as to talk to the stakeholders.
+Managing all the stakeholders for a product can be quite time consuming.
+But there are some techniques that can help the PO to keep an overview.
+
+The PO's superior task is to create a common product vision and to steer the product release.
+They are also responsible for accepting (or not ;) ) the developed tickets from the Development Team.
+As you can see, the Product Owner is a very important and complex role in Scrum and so it’s no wonder that several trainings exist one should participate in before agreeing on this job.
+
+# Scrum Master
+
+{% image blog-scrum2-scrummaster.jpg 30% left %}{% endimage %}
+
+The Scrum Master is normally the most unclear role for people who are new to Scrum.
+The concept is just not existing in other frameworks and therefore the value of this role is often underestimated.
+After all, the Scrum Master is not directly adding development power to the team, so why should a company still invest in an extra employee?
+
+The superior task of a Scrum Master is to protect the Scrum Team!
+And yes, this includes developers and PO.
+One example that most software developers probably understand is the problem of scope creep.
+The developers are working on their feature, when some manager steps into the room and forces someone to take care of an ultra-urgent topic.
+The developer is distracted from his actual task and since it was a manager who told them to go work on something else, they usually do it.
```

Comment:
The developer is distracted from their actual task and since it was a manager who told them to go work on something else, they usually do it.